packages/core/src/components/steps-modal/stories/steps-modal.args.ts

```diff
@@ -55,6 +55,22 @@ export const ModalStoryArgs = {
         category: Category.PROPERTIES,
       },
     },
+    disablePrimary: {
+      control: 'boolean',
+      description:
+        'When true, the primary button is disabled. Note: this only applies to the last step.',
+      table: {
+        category: Category.PROPERTIES,
+      },
+    },
+    disableSecondary: {
+      control: 'boolean',
+      description:
+        'When true, the secondary button is disabled. Note: this only applies to the first step.',
```

Comment:
The name of prop could be more descriptive, like a `disableSecondaryButtonOnFirstStep`

Reply:
fixed at [8f193f5](https://github.com/juntossomosmais/atomium/pull/609/commits/8f193f56a5d7c1ce8a910c727d784eec1b12e5f8)

Reply:
[the same](https://github.com/juntossomosmais/atomium/pull/609#discussion_r1831626796)

Reply:
that was made here [27c24b2](https://github.com/juntossomosmais/atomium/pull/609/commits/27c24b280a5c07b7ff2dee2b77f04f2652eb79ad) i sent it in different commits 😅